packages/apollo-server-core/src/ApolloServer.ts

```diff
@@ -290,6 +297,13 @@ export class ApolloServerBase {
     if (this.engineServiceId) {
       this.engineReportingAgent = new EngineReportingAgent(
         typeof engine === 'object' ? engine : Object.create(null),
+        {
+          schema: this.schema,
+          schemaHash: this.schemaHash,
```

Comment:
Just a thought: a feature that I would want (that would be easy to add) would be for the schema information to be outputted in the responses as an extension. You can see tracing and caching information is response extensions already, but this information, though similar, isn't exposed via extensions, so it feels a bit more opaque. I think we should open it up in case community members want to do things with that schema-awareness themselves :)

Reply:
@zionts I love that idea! The new request pipeline has the `schemaHash` as part of it so it has so many options!

Reply:
This would be especially helpful for publishing your schema without having to specify the tag in the cli, though I understand that we might move to a world of automatically publishing schema on startup